### DISCUSSION TITLE
Recommending based on people I follow but I don't follow anyone

### DISCUSSION THREAD

**Discussion by newnishad** [community/community#15316] [Other Feature Feedback, Questions, & Ideas]
<img width="1122" alt="Screen Shot 2022-04-21 at 4 03 31 PM" src="https://user-images.githubusercontent.com/21108524/164432059-5fe93e22-ee27-4ef8-9585-489ea2e942fd.png">
<img width="418" alt="Screen Shot 2022-04-21 at 4 06 59 PM" src="https://user-images.githubusercontent.com/21108524/164432212-70f50b3a-f923-4b15-a86f-cdc52e61fc15.png">

**Comment by mrjones2014** [comment]
The entire "For You" feed is social media garbage that should be removed. Nobody wants their data to be used this way, algorithmic feeds are toxic.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
